jenkinsfile-runner-tests/test_resources/test_cwp_casc_with_pipeline_library/casc.yml

```diff
@@ -0,0 +1,19 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code Plugin\n\n"
+  numExecutors: 5
+  scmCheckoutRetryCount: 2
+  mode: NORMAL
+
+unclassified:
+  globalLibraries:
+    libraries:
+      - name: "awesome-lib"
+        retriever:
+          legacySCM:
+            scm:
+              git:
+                branches:
+                  - name: "*/master"
```

Comment:
Maybe better a tag to avoid changes in the library affecting these tests?

Reply:
That's a good idea, but unfortunately at the moment there are no tags or branches on that library.